editor/src/components/canvas/controls/select-mode/padding-resize-control.tsx

```diff
@@ -56,9 +56,9 @@ function sizeFromOrientation(orientation: Orientation, desiredSize: Size): Size
 
 export const PaddingResizeControlHoverTimeout: number = 200
 
-const PaddingResizeControlWidth = 2
+const PaddingResizeControlWidth = 4
```

Comment:
Presumably these `const`s should be used in `flex-gap-control.tsx` then if we're aiming to keep the styling of both types of controls in sync (maybe in which case they could be renamed 🤷 )